build_tools/bazel/iree_e2e_generated_test.bzl

```diff
@@ -28,8 +29,9 @@ def iree_e2e_matmul_test(
 
     Args:
         name: Name of the target
-        matmul_src: mlir source file with matmuls to be compiled.
-        matmuls_vmfb: specifies the path to use for the generated IREE module.
+        test_name: Name of the test (e.g., matmuls, conv2ds).
```

Comment:
Why both `name` and `test_name`? Would `test_type` be a better name? Is the extra argument even needed?

Reply:
The flags and naming might differ, but `test_type` sounds good.